kitsune/flagit/jinja2/flagit/includes/flagged_question.html

```diff
@@ -1,27 +1,55 @@
 {% from 'flagit/includes/macros.html' import date_by_user with context %}
-<h3 class="sumo-page-intro">{{ _('Title:') }}</h3>
-<p class="sumo-page-subheading title">{{ object.content_object.title }}</p>
-<h3 class="sumo-page-intro">{{ _('Content:') }}</h3>
-<div class="content">{{ object.content_object.content_parsed }}</div>
-<h3 class="sumo-page-intro">{{ _('Created:') }}</h3>
-<p class="created">
-  {{ date_by_user(object.content_object.created, object.content_object.creator) }}
-</p>
-{% if object.content_object.updated_by %}
-  <h3 class="sumo-page-intro">{{ _('Updated:') }}</h3>
-  <p class="updated">
-    {{ date_by_user(object.content_object.updated, object.content_object.updated_by) }}
-  </p>
-{% endif %}
-<h3 class="sumo-page-intro">{{ _('Flagged:') }}</h3>
-<p class="flagged">
-  {{ date_by_user(object.created, object.creator) }}
-</p>
-<h3 class="sumo-page-intro">{{ _('Take Action:') }}</h3>
-<div class="actions sumo-button-wrap">
-  <a class="sumo-button button-sm" href="{{ object.content_object.get_absolute_url() }}">{{ _('View') }}</a>
-  <a class="sumo-button button-sm edit" href="{{ url('questions.edit_question', object.content_object.id) }}">{{ _('Edit') }}</a>
-  {% if user.has_perm('questions.delete_question') %}
-    <a class="sumo-button button-sm delete" href="{{ url('questions.delete', object.content_object.id) }}">{{ _('Delete') }}</a>
+<div class="flagged-content">
+  <h3 class="flagged-content__heading">{{ _('Title:') }}</h3>
+  <p class="flagged-content__title">{{ object.content_object.title }}</p>
```

Comment:
We shouldn't show the title for answers, right? I wonder if we could simply wrap this with a `{% if object.content_object.title %}...{% endif %}`?

<img width="579" alt="image" src="https://github.com/user-attachments/assets/8e9cab6d-d5ee-43b5-93b7-8cc51f19d15b">